index.html

```diff
@@ -46,5 +46,146 @@ <h3 class="title_2">LET’S CONNECT</h3>
         </li>
       </ul>
     </section>
+    <section class="work">
+      <div class="work_1">
+        <img src="./image\Snapshoot Portfolio.png" alt="Snap_1" />
+        <h2 class="tonic">Tonic</h2>
+        <div class="details">
+          <h3 class="canopy">CANOPY</h3>
+          <div class="dot"></div>
+          <h3 class="back_end">Back End Dev</h3>
+          <div class="dot"></div>
+          <h3 class="a_2015">2015</h3>
+        </div>
+
+        <p class="para_2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="language">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button class="button" type="button" value="see project">see project</button>
+      </div>
+      <div class="work_1">
+        <img src="./image/Snapshoot Portfolio (1).png" alt="Snap_2" />
+        <h2 class="tonic">Multi-Post Stories</h2>
+        <div class="details">
+          <h3 class="canopy">CANOPY</h3>
+          <div class="dot"></div>
+          <h3 class="back_end">Back End Dev</h3>
+          <div class="dot"></div>
+          <h3 class="a_2015">2015</h3>
+        </div>
```

Comment:
- [ ] For semantic reasons, do you think you can make a `ul` element with each nested element as `li`? This is conventional to have a similar list of group of same items enumerated with an `ul` tag. It even makes styling easier.